### PR TITLE
Added plugins page

### PR DESCRIPTION
**Pull request by dimitrioskarvounaris**

### CODE REVIEW

**Comment by octoper** [comment]
@AlexMartinFR

**Comment by alexmartinfr** [comment]
Nice work, thanks for this contribution!
(I'll move it to the Guide section, it's a bit advanced for the "Getting Started" one)

File: source/docs/plugins.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+```bash
+composer require fzaninotto/faker
+```
+
+All plugin files reside inside the `src/` folder. Let's assume we want to provide a `faker()` method
+for our tests. We create a file `InteractsWithFaker.php` with the following code:
+
+```php
+if (!function_exists('faker')) {
+    /** @phpstan-ignore-next-line */
```
Comment (nunomaduro) [inline source/docs/plugins.md:38]: This should be removed.

**Comment by alexmartinfr** [comment]
I should have let you do that yourself, @dimitrioskarvounaris , I had to move your file to a new one and now the commit history don't mention you anymore.

I'm sorry for that, thanks again for your great contribution!

**Comment by dimitrioskarvounaris** [comment]
You can set the author of a commit to someone else when you commit something ;)

**Comment by alexmartinfr** [comment]
I don't find the option on GitHub, though, I guess you have to do it on the CLI.
I'll definitely handle that differently next time :)

**Pull request merged**